### PR TITLE
updates all the incorrect tocs AND the maintenance instructions

### PR DESCRIPTION
**Pull request by kilbergr**
# [[User Submitted] mariadb.east.eclkc/Disk Check - /var/backups is CRITICAL](https://ocio-jira.acf.hhs.gov/browse/OHSH-131)

Changes proposed in this pull request:

- Updates maintenance page with a lot more instructions for how to do various disk/volume changes
- Updates various TOCs in docs that were inaccurate.

### CODE REVIEW

File: docs/runbooks/maintenance.md [hunk 1]
```diff
@@ -2,20 +2,101 @@
...
+* [Update NodeJS to version 12](#update-nodejs-to-version-12)
+* [Updating packages on Ubuntu](#updating-packages-on-ubuntu)
+* [Updating pacakages on Centos](#updating-pacakages-on-centos)
+* [Cleaning up old kernels on Centos](#cleaning-up-old-kernels-on-centos)
+
+<!-- Regenerate with "pre-commit run -a markdown-toc" -->
+
+<!-- tocstop -->
+
+## Create a new volume
```
Comment (eeeady) [inline docs/runbooks/maintenance.md:26]: Hah! I think something we both missed is when you create the new volume enter in a number greater than the size previously.

Also note you cannot downsize a volume from a snapshot.

File: docs/runbooks/maintenance.md [hunk 2]
```diff
@@ -2,20 +2,101 @@
...
+   1. If the mounted disk does not show the correct size after you mount it (examine by running `df -h`), run `[sudo] xfs_growfs /var/<MOUNT>`. Run `df -h` again to confirm it is the correct size.
+
 1. Add a `ext4` filesystem on top. Note, you may want an xfs file system but that will use different utlities.
    1. use `mkfs.ext4` on the partition device you created.
-1. Add a descriptive label to the new device to identify it. (eg. if you are creating a disk to mount to `/var/backups' label it 'backups'.)
-1. Add new device mount using the label to /etc/fstab so that it will automatically mounted on a reboot. See [this linuxhint article](https://linuxhint.com/mount_partition_uuid_label_linux/ ) for further instructions.
+1. [Add a label to a disk](#add-a-label-to-a-disk) to give it a human-readable identifier.
+1. If you're switching a volume for a copy of a larger size:
+   1. [Sync the backup](#sync-a-backup) then [unmount](#unmount-a-disk) both the current volume and replacement volume. 
+   1. Then [mount the replacement volume](#mount-a-disk) to the original disk. XXXXXXXXX THIS PART.
```
Comment (eeeady) [inline docs/runbooks/maintenance.md:58]: did you mean to delete or replace this line?

**Review by eeeady — CHANGES_REQUESTED** [review]
I think we are missing some info about making sure the new volume is bigger.

Additionally there was a line that looks like you wanted to replace? If not maybe take out the comment.

File: docs/runbooks/maintenance.md (hunk 2 above)
Comment (kilbergr) [reply docs/runbooks/maintenance.md:58]: No--I wrote a note about it in the PR:

PLEASE HELP!
There's a line (https://github.com/OHS-Hosting-Infrastructure/infrastructure/compare/rek-131-increase-disk-space?expand=1#diff-85e9d0513e5021b6c54bd48397c4a366d3498e677637372ab2014a41b78b737fR58) marked "XXXXXXXXX THIS PART. " where I am still a little confused what's happening. Can you check that what I've written is accurate and fully represents the action needed at that part? I'm still a little confused why we sync to the second disk (the temp one) and then mount to the first disk anyway (the preexisting one).

Comment (eeeady) [reply docs/runbooks/maintenance.md:58]: Got it! I'll take a look.

Comment (eeeady) [reply docs/runbooks/maintenance.md:58]: OOH I SEE... lemme suggest a change.

Comment (eeeady) [reply docs/runbooks/maintenance.md:58]:
```suggestion
   1. Then [mount the replacement volume](#mount-a-disk) to the original mount path in the filesystem.
```

**Review by jerels — APPROVED** [review]
lgtm!!

**Pull request merged**